hieradata/staging.yaml

```diff
@@ -20,7 +20,7 @@ govuk::apps::email_alert_api::govdelivery_account_code: 'UKGOVUK'
 govuk::apps::email_alert_api::govdelivery_hostname: 'stage-api.govdelivery.com'
 govuk::apps::email_alert_api::govdelivery_public_hostname: 'stage-public.govdelivery.com'
 govuk::apps::email_alert_api::govuk_notify_base_url: 'https://api.staging-notify.works'
```

Comment:
Should we also stop using Notify's staging environment now?

Reply:
I thought we might want to run some more load tests there with digests but let me know if not and I can remove it.

Reply:
I'm just thinking that we can't change the template ID or API key without also changing back to use the production environment because otherwise they won't match.

I suspect we won't need to do any more load testing on staging, so I reckon we can stop using it.

Reply:
Cool, I've removed it.